lib/absinthe/type/extensions.ex

```diff
@@ -0,0 +1,112 @@
+defmodule Absinthe.Type.Extensions do
+  use Absinthe.Schema.Notation
+
+  @moduledoc """
+  This module contains additional data types.
+
+  To use: `import_types Absinthe.Type.Extensions`.
+  """
+
+  scalar :datetime, name: "DateTime" do
+    description """
+    The `DateTime` scalar type represents a date and time in the UTC
+    timezone. The DateTime appears in a JSON response as an ISO8601 formatted
+    string, including UTC timezone ("Z").
+    """
+
+    serialize &DateTime.to_iso8601/1
+    parse parse_with([Absinthe.Blueprint.Input.DateTime], &parse_datetime/1)
```

Comment:
Got it! I should have better reviewed when I moved the code from builtins to extensions :)

I'll update soon.